### PR TITLE
JWT issuer claim configurable

### PR DESCRIPTION
**Pull request by azdagron**

### CODE REVIEW

File: cmd/spire-server/cli/run/run.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ type serverConfig struct {
 	BindPort            int                `hcl:"bind_port"`
 	CASubject           *caSubjectConfig   `hcl:"ca_subject"`
 	CATTL               string             `hcl:"ca_ttl"`
+	JWTIssuer           string             `hcl:"jwt_issuer"`
```
Comment (evan2645) [inline cmd/spire-server/cli/run/run.go]: nit: alphabetical ordering?

File: cmd/spire-server/cli/run/run.go [hunk 2]
```diff
@@ -323,6 +324,8 @@ func newServerConfig(c *config) (*server.Config, error) {
 		sc.CATTL = ttl
 	}
 
+	sc.JWTIssuer = c.Server.JWTIssuer
```
Comment (evan2645) [inline cmd/spire-server/cli/run/run.go]: Assuming it's not too difficult to do, would be good to get a small test case to ensure this remains wired in. I've been bitten (and seen others get bitten) in the past during refactors etc by missing these one-liners.

File: pkg/agent/endpoints/workload/handler_test.go [hunk 3]
```diff
@@ -558,7 +558,9 @@ func (s *HandlerTestSuite) TestValidateJWTSVID() {
 	})
 	s.Require().NoError(err)
 
-	jwtSigner := jwtsvid.NewSigner(jwtsvid.SignerConfig{})
+	jwtSigner := jwtsvid.NewSigner(jwtsvid.SignerConfig{
+		Issuer: "issuer",
```
Comment (evan2645) [inline pkg/agent/endpoints/workload/handler_test.go]: Do we want a test case here where issuer is not set? Just a thought, not a blocker

File: cmd/spire-server/cli/run/run.go (hunk 1 above)
Comment (azdagron) [reply cmd/spire-server/cli/run/run.go]: 👍

File: cmd/spire-server/cli/run/run.go (hunk 2 above)
Comment (azdagron) [reply cmd/spire-server/cli/run/run.go]: Oh, yep. Tests. Will do!

File: pkg/agent/endpoints/workload/handler_test.go (hunk 3 above)
Comment (azdagron) [reply pkg/agent/endpoints/workload/handler_test.go]: Sure, i can add one real quick.

**Pull request merged**